package.json

```diff
@@ -1,53 +1,78 @@
 {
-    "name": "tera",
-    "displayName": "Tera Syntax Highlighting",
-    "description": "Tera Template Syntax Highlighting",
-    "version": "0.0.2",
-    "publisher": "karunamurti",
-    "engines": {
-        "vscode": "^1.19.0"
+  "name": "tera",
+  "displayName": "Tera",
+  "description": "A powerful, easy-to-use template language based on Rust, inspired by Jinja2 and Django Templates.",
+  "version": "0.0.2",
+  "publisher": "karunamurti",
+  "engines": {
+    "vscode": "^1.19.0"
+  },
+  "categories": [
+    "Programming Languages",
+    "Formatters"
```

Comment:
Added `Formatters`, and changed it to `Programming Languages` (seems to be the new default)